apps/api/src/app/feature-flags/services/feature-flags.service.ts

```diff
@@ -49,7 +49,8 @@ export class FeatureFlagsService {
     }
   }
 
-  public async get<T>(key: FeatureFlagKey, context: IFeatureFlagContext, defaultValue: T): Promise<T> {
-    return await this.service.get(key, context, defaultValue);
+  public async get<T>(key: FeatureFlagKey, defaultValue: T, context: IFeatureFlagContext): Promise<T> {
```

Comment:
swapped the order because I found it was a clearer API as the key and the default value are related and context might be optional in a further decision.